.circleci/config.yml

```diff
@@ -659,6 +671,20 @@ jobs:
             docker push ${DOCKERHUB_REPO:?}:${CIRCLE_BRANCH:?}
       - *fail_notification
 
+  docker_push_latest:
+    executor: infrastructure_container_stable
+    <<: *infrastructure_config
+    steps:
+      - checkout
+      - *setup_secrets
+      - *setup_remote_docker
+      - run:
+          name: Build and push Docker image to DockerHub
+          command: |
+            docker login -u $DOCKER_USER -p $DOCKER_PASS
+            docker tag ${DOCKERHUB_REPO:?}:${CIRCLE_TAG:?} ${DOCKERHUB_REPO:?}:latest
```

Comment:
Who guarantees that Docker image tag `${CIRCLE_TAG:?}` present on this host? I am concerned of this intermittently failing (and hence having surprise at release).

Reply:
OK, Agree. The docker cache should cover the rebuild time hopefully